bc_obps/registration/fixtures/mock/facility.json

```diff
@@ -334,5 +334,271 @@
       "latitude_of_largest_emissions": 43.5,
       "longitude_of_largest_emissions": -123.5
     }
+  },
+  {
+    "model": "registration.facility",
+    "pk": "67fd8288-422b-43d3-a9b1-e4ddc54e2139",
+    "fields": {
+      "created_by": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "created_at": "2024-06-05T23:18:07.664Z",
+      "name": "Facility 25",
+      "type": "Single Facility",
+      "swrs_facility_id": 1025,
+      "bcghg_id": "13219990028",
+      "latitude_of_largest_emissions": 43.5,
+      "longitude_of_largest_emissions": -123.5
+    }
+  },
+  {
+    "model": "registration.facility",
+    "pk": "9f7b0848-021e-4d08-9852-10524c4e5456",
+    "fields": {
+      "created_by": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "created_at": "2024-06-05T23:18:07.664Z",
+      "name": "Facility 26",
+      "type": "Single Facility",
+      "swrs_facility_id": 1026,
+      "bcghg_id": "13219990029",
```

Comment:
Can we remove bcghg_id from some of the facilities? This will make it easier for testers to assign them